app/scripts/controllers/createClientBuild.js

```diff
@@ -98,21 +105,26 @@ angular.module('openshiftConsole')
                 {
                   name: 'BUILD_CONFIG',
                   value: clientConfig.buildType
+                },
+                {
+                  name: 'PLATFORM',
+                  value: clientConfig.buildPlatform
```

Comment:
Adding this all the time, i don;t see the harm. @odra Shouldn't be a an issue should it?

Reply:
I think it is fine, we just need to add this information into the cordova docs.